### PR TITLE
Use new abi coder pragma

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: test/libsolidity/gasTests/abiv2.sol [hunk 1]
```diff
@@ -1,4 +1,4 @@
-pragma experimental ABIEncoderV2;
+pragma abicoder               v2;
```
Comment (chriseth) [inline test/libsolidity/gasTests/abiv2.sol:1]: The purpose of the spaces is that we do not have to update the error locations.

**Comment by axic** [comment]
Tests are failing 😬

**Comment by leonardoalt** [comment]
Looks good to me, not sure why tests are failing

**Review by mijovic — APPROVED** [review]
Good to merge if all tests are passing~

**Comment by axic** [comment]
There are still a bunch of files using the old one:
```
syntaxTests/specialFunctions/abidecode/abi_decode_memory_v2.sol:pragma experimental "ABIEncoderV2";
syntaxTests/abiEncoder/selected_twice_v2.sol:pragma experimental ABIEncoderV2;
syntaxTests/abiEncoder/conflicting_settings_reverse.sol:pragma experimental ABIEncoderV2;
syntaxTests/abiEncoder/same_setting_twice.sol:pragma experimental ABIEncoderV2;
semanticTests//array/copying/array_of_struct_calldata_to_memory.sol:pragma experimental ABIEncoderV2;
semanticTests//array/copying/array_of_structs_containing_arrays_calldata_to_memory.sol:pragma experimental ABIEncoderV2;
semanticTests//array/copying/array_nested_calldata_to_memory.sol:pragma experimental ABIEncoderV2;
semanticTests//structs/calldata/calldata_struct_to_memory_tuple_assignment.sol:pragma experimental ABIEncoderV2;
semanticTests//structs/calldata/calldata_struct_as_argument_of_lib_function.sol:pragma experimental ABIEncoderV2;
semanticTests//structs/calldata/calldata_struct_with_array_to_memory.sol:pragma experimental ABIEncoderV2;
semanticTests//structs/calldata/calldata_nested_structs.sol:pragma experimental ABIEncoderV2;
semanticTests//structs/calldata/calldata_struct_as_memory_argument.sol:pragma experimental ABIEncoderV2;
semanticTests//structs/calldata/calldata_struct_with_nested_array_to_memory.sol:pragma experimental ABIEncoderV2;
```

**Comment by chriseth** [comment]
Looks like they were merged while working on this.

**Pull request merged**